recipes/node.rb

```diff
@@ -290,7 +290,7 @@
 
   execute 'Wait for API to become available before starting Node component' do
     command "[[ $(curl --silent --tlsv1.2 --max-time 2 ${MASTER_URL}/healthz/ready --cacert #{node['cookbook-openshift3']['openshift_node_config_dir']}/ca.crt) =~ \"ok\" ]]"
-    environment 'MASTER_URL' => node['cookbook-openshift3']['openshift_HA'] ? node['cookbook-openshift3']['openshift_master_api_url'] : "https://#{node['cookbook-openshift3']['openshift_common_api_hostname']}:#{node['cookbook-openshift3']['openshift_master_api_port']}"
```

Comment:
We should ignore the cert validation on nodes due to logic when using named_certificates